.github/ISSUE_TEMPLATE/unreproducible_package.md

```diff
@@ -7,22 +7,62 @@ assignees: ''
 
 ---
 
-Building this package twice does not produce the bit-by-bit identical result each time, making it harder to detect CI breaches. You can read more about this at https://reproducible-builds.org/ .
+Building this package multiple times does not yield bit-by-bit identical
+results, complicating the detection of Continuous Integration (CI) breaches. For
+more information on this issue, visit
+[reproducible-builds.org](https://reproducible-builds.org/).
 
-Fixing bit-by-bit reproducibility also has additional advantages, such as avoiding hard-to-reproduce bugs, making content-addressed storage more effective and reducing rebuilds in such systems.
+Fixing bit-by-bit reproducibility also has additional advantages, such as
+avoiding hard-to-reproduce bugs, making content-addressed storage more effective
+and reducing rebuilds in such systems.
 
 ### Steps To Reproduce
 
+In the following steps, replace `<package>` with the canonical name of the package.
```

Comment:
Ideally the person creating an issue using this template should replace <package> with the package name, could you add instructions for them to do that?

Reply:
But that's precisely the instruction, is it missing something?

Reply:
Something like this:

```
<!-- Dear reporter, replace <package> with the canonical name of the package
you create this issue for. -->
```

On top of the issue ?

Reply:
Right now it's not super obvious this is an instruction for the reporter (rather for who-ever is reading the issue) - indeed I think adding something like 'Dear reporter' (either at the top or here) would go a long way

Reply:
Let me quickly do that :)

Reply:
Done